debug/tests/esm.html

```diff
@@ -1,19 +0,0 @@
-<html>
-  <head>
-    <link rel="stylesheet" href="../../dist/leaflet.css" />
-    <link rel="stylesheet" href="../css/screen.css" />
-  </head>
-  <body>
-    <div id="map"></div>
-    <script type="module">
-      import L from '../../dist/leaflet-src.esm.js';
-
-      var map = L.map('map', {
-        center: [39.84, -96.591],
-        zoom: 4,
-      });
-
-      L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png').addTo(map);
```

Comment:
why deleting this file? Only the import path should be changed

Reply:
Yeah, I guess there is no harm in keeping this around. Restored the file and changed the path.